modules/schema/src/main/java/org/apache/ignite/internal/schema/registry/SchemaRegistryImpl.java

```diff
@@ -148,12 +149,11 @@ public Row resolve(BinaryRow row) {
     public Collection<Row> resolve(Collection<BinaryRow> rows) {
         final SchemaDescriptor curSchema = waitLatestSchema();
 
-        return rows.stream().map(row -> resolveInternal(row, curSchema)).collect(toList());
+        return rows.stream().filter(Objects::nonNull).map(row -> resolveInternal(row, curSchema)).collect(toList());
```

Comment:
Please, rewrite this line with code that does not use Stream API. This code potentially is on a hot path.